clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -72,6 +72,9 @@ for P4Runtime switches:
   get-counter SWITCH CTR_TBL CTR_FLOW   gets indirect counter table entry
   get-direct-counter SWITCH CTR_TBL CTR_FLOW   gets direct counter table entry
   reset-counter SWITCH CTR_TBL CTR_FLOW resets counter table entry
+
+  get-meter SWITCH MTR_TBL MTR_FLOW     gets packet mod meter table entry
```

Comment:
Should we be calling these `get-packet-mod-meter` ?

Reply:
Fixed

Reply:
I don't think this was fixed. Only the next one was

Reply:
Now fixed